cicd/buildspec-deploy.yml

```
@@ -199,6 +199,8 @@ phases:
         echo 'release-notes/deactivated|/explore' >> ./redirects.txt
         echo 'go-live|/onboarding/request-prod-access' >> ./redirects.txt
         echo 'oauth|/explore' >> ./redirects.txt
+        echo 'explore/authorization/docs/authorization-code|explore?auth=acg' >> ./redirects.txt
+        echo 'explore/authorization/docs/client-credentials|explore?auth=ccg' >> ./redirects.txt
```

These need to start with the / on the target url.
```suggestion
        echo 'explore/authorization/docs/authorization-code|/explore?auth=acg' >> ./redirects.txt
        echo 'explore/authorization/docs/client-credentials|/explore?auth=ccg' >> ./redirects.txt
```

---

Good catch!